From 1e7cecb2d70131cca4f63b1931d6ed4989096986 Mon Sep 17 00:00:00 2001
From: "Tao, Jing" <jing.tao@intel.com>
Date: Mon, 27 Oct 2014 13:40:51 +0800
Subject: [PATCH 113/117] ov5693-ecs: add 5M resolution for video mode

The SDV mode needs to choose this resolution.

Change-Id: I5dd4458a9f0c973865e6fead1d05477f1deed151
Tracked-On: https://bz01p-vied.ir.intel.com/show_bug.cgi?id=4319
Signed-off-by: Tao, Jing <jing.tao@intel.com>
---
 drivers/external_drivers/camera/drivers/media/i2c/ov5693.h | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h b/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h
index 5cfed50..2ec30e3 100755
--- a/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h
+++ b/drivers/external_drivers/camera/drivers/media/i2c/ov5693.h
@@ -947,6 +947,20 @@ struct ov5693_resolution ov5693_res_video[] = {
 		.bin_mode = 0,
 		.regs = ov5693_2592x1456_30fps,
 	},
+	{
+		.desc = "ov5693_5M_30fps",
+		.width = 2592,
+		.height = 1944,
+		.pix_clk_freq = 160,
+		.fps = 30,
+		.used = 0,
+		.pixels_per_line = 2688,
+		.lines_per_frame = 1984,
+		.bin_factor_x = 1,
+		.bin_factor_y = 1,
+		.bin_mode = 0,
+		.regs = ov5693_2592x1944_30fps,
+	},
 };
 #define N_RES_VIDEO (ARRAY_SIZE(ov5693_res_video))
 
-- 
1.9.3

